test/quantization/test_numeric_suite_fx.py

```diff
@@ -1338,7 +1345,7 @@ def forward(self, x):
         # test shadowed activations
 
         node_type_to_io_type_map = get_node_type_to_io_type_map()
-        node_type_to_io_type_map['funs_io_type_fp32'].add(_wrapped_hardswish)
+        node_type_to_io_type_map['funs_io_type_fp32'].add(_wrapped_hardswish_fp16)
```

Comment:
_wrapped_hardswich_fp16 seem to have fp16 input and output type, why are we adding this to funs_io_type_fp32?

Reply:
fixed

Reply:
sorry, fix upcoming, but ghstack is having some issues, please ignore previous comment